projects/systelab-components/src/lib/combobox/autocomplete/autocomplete-combobox.component.html

```diff
@@ -2,9 +2,9 @@
      class="dropdown slab-combobox d-flex w-100"
      [ngClass]="{'disabled': isDisabled}">
     <div #dropdowntoogle class="slab-flex-1 d-flex dropdown-toggle slab-dropdown-toogle" id="{{comboId}}" data-toggle="dropdown">
-        <input #input type="text" class="slab-flex-1 form-control d-flex slab-combo-input"
+        <input #input type="text" class="slab-flex-1 form-control d-flex slab-combo-input" keyup-debounce
                [disabled]="isDisabled"
-               [(ngModel)]="fieldToShow" (keyup)="doSearch($event)"
+               [(ngModel)]="fieldToShow" (keyupDebounced)="doSearch($event)"
```

Comment:
I would suggest to add the debounce time of the directive as a public property of the auto complete combo box, so that it can be set by anyone using the combo box.
Specially considering that maybe some client applications need to be able to reduce this debouncing in E2E tests to reduce unnecessary wait times.

Reply:
done